modules/nw-mutual-tls-auth.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * ingress/ingress-operator.adoc
+
+[id=nw-mutual-tls-auth_{context}]
+= Configuring Mutual TLS Authentication
+
+You can configure the Ingress Controller to enable mutual clientTLS(mTLS) authentication by setting `spec.clientTLS`. The clientTLS configures the Ingress controller to verify client certificates (CAs), certificate revocation list (CRL), and/or any certificate subject filters. Requests that do not provide valid certificates are rejected.
```

Comment:
The description might be clearer if it avoided mixing "client TLS" and "mutual TLS".  The Route API allows configuring server TLS, this new IngressController API allows configuring client TLS, and client and server TLS together constitute mutual TLS.  

The user can use the API to configure a CA certificate that is used to sign client certificates.  The operator downloads any associated CRL (the user doesn't specify the CRL directly).  
```suggestion
You can configure the Ingress Controller to enable mutual TLS (mTLS) authentication by setting `spec.clientTLS`. The `clientTLS` field specifies configuration for the Ingress Controller to verify client certificates.  This configuration includes the CA certificate that is used to sign client certificates, as well as an optional list of certificate subject filters. Requests that do not provide valid certificates are rejected. If the CA certificate specifies an X509v3 certificate revocation list (CRL) distribution point, the Ingress Controller downloads and enforces the CRL.
```